js/main.js

```diff
@@ -0,0 +1,412 @@
+import keyboardKeys from './keyboardKeys.js';
+
+// Creating keyboard elements
+const body = document.querySelector('body');
+
+const divWrap = document.createElement('div');
```

Comment:
creata a file templateHelper.js

add new function there, that can build any common html tag and set up them.

```
export const buildHtmlElement = ({tagName, classList, attrs = {}}) => {
  const newElement =  document.createElement(tagName);
  classList && newElement.classList.add(...classList);
  Object.keys(attrs).forEach((key) => newElement.setAttribute(key, attrs[key]));
  // there can be any other element manipulation
  return newElement;
}; 
```

reuse it when you create html elemets, i.e. 
`const textArea = buildHtmlElement({tagName:'textarea',attrs:{id:'textarea-id',type:'textarea'}})`